client/my-sites/sites/sites.jsx

```diff
@@ -47,7 +47,7 @@ export const Sites = React.createClass( {
 
 		// Filter out sites with no upgrades on particular routes
 		if ( /^\/domains/.test( path ) || /^\/plans/.test( this.props.sourcePath ) ) {
-			return this.props.isSiteUpgradeable( site.ID );
+			return site.isSiteUpgradeable !== false;
```

Comment:
What does state look like in that case? I'd also be fine with the above and adding a clarifying comment so folks don't try and simplify the check.